lib/build_manager.py

```diff
@@ -43,9 +43,9 @@ def _build_packages(self, distro, packages):
         """
```

Comment:
I disagree that 'lsb' does not add useful information. It means 'Linux Standard Base' and it is a Linux distribution unique ID. You could rename to lsb_id if you prefer.

"Name" is not a standardized value. Why we'd use "CentOS" instead of "Cent OS", for instance?